### PR TITLE
feat: Spiffe and Spire website redesign

### PR DESCRIPTION
**Pull request by thisisobate**
**Todos**
- [x] Create layout/structure using plain HTML
- [x] Style page
- [x] Implement mobile responsiveness
- [x] Convert HTML to Hugo templates
- [x] Split file into components
- [x] Convert used-by section into a marquee
- [x] Add more logos to used-by section
- [x] Add spiffe logo to navbar
- [x] Add the news section at the top of the page
- [x] Add turtle image to the background
- [x] Make all the used-by logo to be the same width and height
- [x] Clean code
- [x] make users logos scroll infinitely
- [x] Change spiffe and spire to uppercase in overview page
- [x] Sort all the logos
- [x] Add spacing to logos
- [x] rename integrations to Integrations and implementations
- [x] make logos to be clickable
**~~Follow-up PR~~**
- [x] Fix title alignment in Spiffe and Spire documentation page.

### CODE REVIEW

**Comment by thisisobate** [comment]
cc: @nate-double-u

**Review by sanderson042 — COMMENTED** [review]
Hi Uche, thanks for your continued work on this. Here's a few comments:
* In the Overview section, I see "Spiffe" and "Spire"  The preferred forms are all caps: “SPIFFE” and “SPIRE”.
* The horizontal scroll of the logos in the Used By section is maybe a little too fast. It would be nice for it to wrap around in an infinite loop rather than start over after Arm. It’s hard to see the Arm logo. On second thought, maybe a static responsive group would be better.
* This item noted before is still an issue: “The alignment of the top titles on [doc landing page](https://deploy-preview-264--spiffe.netlify.app/docs/latest/spiffe-about/overview/) and the [download page](https://deploy-preview-264--spiffe.netlify.app/downloads/) were unintentionally affected by the landing page update.” On both, the vertical spacing is slightly too big. On Get SPIRE, the heading is centered rather than flush left.
* This item in the previous review still seems a bit odd to me: “Under “Benefits”, the text in the cards on the left is hard to read. Uche had meant them more as a design element.” Maybe the text in the cards could be made bigger and intentionally cut off so it looks like each card is the top of a longer document.
* Please check the font size for all the headings. For example "Used By" and "Overview" seem to have slightly different font sizes but I think they should be the same.
* The headings should use a consistent capitalization scheme. The quickest fix is to capitalize all three words in "Key Use cases".

Here's the previous list of suggestions from the SPIFFE Community meeting notes on December 15, 2022: 

> Suggestions and comments for Uche discussed today (he’ll be on vacation from late Dec. to mid Jan.):
> 	◦	In the “Used By” area, there are too many logos to show in the current static format. Uche will make the logos auto-scroll horizontally. 
> 	◦	In the “Used By” area, logo sizes vary widely, especially on mobile devices. 
> 	◦	Uche received the turtle graphics from Evan but wasn’t sure how to work them in. Evan showed an example with them in the background that Uche may try to implement.
> 	◦	The alignment of the top titles on [doc landing page](https://deploy-preview-264--spiffe.netlify.app/docs/latest/spiffe-about/overview/) and the [download page](https://deploy-preview-264--spiffe.netlify.app/downloads/) were unintentionally affected by the landing page update. Uche plans to investigate.
> 	◦	Move the bar that indicates CNCF graduation closer to or at the top.
> 	◦	Under “Benefits”, the text in the cards on the left is hard to read. Uche had meant them more as a design element. But he may experiment with the colors to see if that helps.

**Comment by thisisobate** [comment]
Thanks for the reviews @sanderson042 

> “The alignment of the top titles on [doc landing page](https://deploy-preview-264--spiffe.netlify.app/docs/latest/spiffe-about/overview/) and the [download page](https://deploy-preview-264--spiffe.netlify.app/downloads/) were unintentionally affected by the landing page update.”

Can you post a screenshot of what you mean in this thread?

**Comment by thisisobate** [comment]
@sanderson042 I have made all the necessary changes. 
Kindly review.
Thanks!

File: layouts/partials/home/hero.html [hunk 1]
```diff
@@ -2,42 +2,27 @@
...
-        Secure Production Identity Framework for Everyone
-      </p>
-      <p class="subtitle is-size-3 is-size-5-mobile has-text-weight-light">
-        {{ $description }}
-      </p>
+  <div class="hero-body is-flex is-align-items-center is-columns-centered p-3 flex-column-mobile-reverse">
+    <div class="">
+      <div class="text-container">
+        <h4 class="title is-2 is-size-4-mobile">Universal identity control plane for distributed systems</h4>
+        <p class="subtitle is-5 subtitle-text">Spiffe and SPIRE provide strongly attested, cryptographic identities to workloads across a wide variety of platforms</p>
```
Comment (mchurichi) [inline layouts/partials/home/hero.html:12]: This should read `SPIFFE`, all uppercase.

Comment (thisisobate) [reply layouts/partials/home/hero.html:12]: Alright. Would fix up now

**Comment by nate-double-u** [comment]
I'd suggest squashing this PR at this point @thisisobate, may help with some of the CI issues you're running into

**Comment by thisisobate** [comment]
I'm afraid squashing commits won't fix the problem; it's more complicated than that. 
Thankfully, the maintainers are already looking into this.

**Comment by sanderson042** [comment]
> Thanks for the reviews @sanderson042
> 
> > “The alignment of the top titles on [doc 

landing page](https://deploy-preview-264--spiffe.netlify.app/docs/latest/spiffe-about/overview/) and the [download page](https://deploy-preview-264--spiffe.netlify.app/downloads/) were unintentionally affected by the landing page update.”
> 
> Can you post a screenshot of what you mean in this thread?

Hi Uche,

Thank you very much for addressing almost all of the things that I noticed! In particular, the text quotes look pretty good now.

What's left are the alignment issues on the Documentation and Get SPIRE pages linked at the top. I've attached four screenshots showing the spacing issues. The thin line on the documentation landing page is not a big problem but the vertical spacing might be good to fix, unless it was intentional. On the Get SPIRE page, the centered title seems inconsistent with the rest of the site and the top has a bit too much vertical spacing.

I noticed that the SPIFFE logo is always in the same place in the upper left on your PR. That is an improvement over the original version, so thanks for that.

Screen Shot - Documentation ORIGINAL
Screen Shot - Documentation NEW
Screen Shot - Get SPIRE ORIGINAL
Screen Shot - Get SPIRE NEW

**Review by anvega — APPROVED** [review]

**Review by sanderson042 — APPROVED** [review]
Hi Uche,

I'm approving this PR. Below I've listed some minor issues that might be good to fix in the future, but for now, this looks good to me. Thank you for your hard work on this and patience with the review process!

Things to fix in a later PR:
(The first two were discussed in the Feb 23, 2023 SPIRE community meeting.)
* In the docs/latest/spiff-about/overview/ page under "Which tools implement SPIFFE?", add blurbs and fix external links to the tools. Currently each tool links to the top of the same page, which is a little confusing.
* On the main landing page under "Benefits," make the three quotes link to a new testimonials page or testimonials on the CNCF site. @umairmkhan and @sanderson042 to work with Uche on this. 
* This is very nitpicky, but for cosmetic reasons the rectangular quotes just mentioned maybe could be offset a little like earlier versions of this page such as in a high, low, medium sequence: :arrow_double_up: :arrow_double_down: ⏹️

**Pull request merged**